index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Apple</title>
+</head>
+
+<body>
+    <main id="core">
+        <section id="core-section">
+            <nav id="top-bar">
+                <a class="nav-links" href="#"><img id="nav-logo" src="assets/apple.svg" alt="Apple Logo">
+                </a>
+                <a class="nav-links" href="#">
+                    Store</a>
+                <a class="nav-links" href="#">
+                    Mac</a>
+                <a class="nav-links" href="#">
+                    iPod</a>
+                <a class="nav-links" href="#">
+                    iPhone</a>
+                <a class="nav-links" href="#">
+                    iPad</a>
+                <a class="nav-links" href="#">
+                    iTunes</a>
+                <a class="nav-links" href="#">
+                    Support</a>
+                <div class="nav-search">
+                    <input type="text">
+                </div>
+            </nav>
+            <header id="core-header">
+                <a href="#" id="one" class="core-links">
+                What will your verse be? 
+                </a>
+                <a href="#"  id="two" class="core-links">
+                Everyone has something to share. See how people are using </br> iPad in their unique ways to add to the world’s story.  
+                </a>
+                <a href="#"  id="three" class="core-links">
+                 Watch the "Your verse" film and explore the stories
+                </a>
+            </header>
+        </section>
+        <aside id="relevant-links">
+            <a href="#" class="aside-links">
+                <img src="assets/promo_iphone5s.jpg" alt="Apple promotional img" class="figure-images">
+            </a>
+            <a href="#" class="aside-links">
+                <img src="assets/promo_iphone5c.jpg" alt="Apple promotional img" class="figure-images">
+            </a>
+            <a href="#" class="aside-links">
+                <img src="assets/promo_30_years.jpg" alt="Apple promotional img" class="figure-images">
+            </a>
+            <a href="#" class="aside-links">
+                <img src="assets/promo_supplier_responsibility.jpg" alt="Apple promotional img" class="figure-images">
+            </a>
+        </aside>
+    </main>
+    <footer id="second-links">
+        <div class="bottom-links"> 
+            <p> 
+            Shop the <a href="#"> Apple Online Store </a> (1-800-MY-APPLE), visit an <a href="#">Apple Retail Store</a>, or find a <a href="#">reseller </a>.
```

Comment:
Please, make the footer look closer to the original.

### ORIGINAL
![original](https://user-images.githubusercontent.com/34820784/64291125-af3b7000-cf67-11e9-9200-ec76b2cdfe89.png)



### YOUR PROJECT
![current](https://user-images.githubusercontent.com/34820784/64291047-9337ce80-cf67-11e9-8357-abebb446a209.png)

Reply:
Oh, for some reason I had errors when pushing, resulting in  not up to date files in the remote repo, its fixed now.